wolox-books/src/router.js

```diff
@@ -9,7 +9,7 @@ export default new Router({
     {
       path: '/',
       name: routes.home,
```

Comment:
This shouldn't be a view, but remember to always match the name of the route with the view.